.pylintrc

```diff
@@ -10,3 +10,8 @@ disable=
     too-many-locals,
     # handled by black
     format
+
+[SIMILARITIES]
+ignore-comments=no
+ignore-docstrings=yes
+ignore-imports=yes
```

Comment:
doesn't duplicate docstrings and imports suggest something could be done better?

Reply:
In principle, yes, but especially imports can lead to false positives. Will change the docstrings, though.

Reply:
The default of 4 lines unfortunately leads to a lot of false postives for duplicate docstrings (especially because of the Sphinx-lines such as `Raises` etc.)

Reply:
cool all good then